dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -116,5 +117,136 @@ def DALIRawIterator():
     return _dali_tf
 
 
+def _get_tf_minor_version():
+  return tf.__version__.split('.')[1]
+
+
+if _get_tf_minor_version() in {'13', '14'}:
+  from tensorflow.python.framework import ops
+  from tensorflow.python.data.ops import dataset_ops
+  from tensorflow.python.data.util import structure
+  import functools
+
+  class _DALIDatasetV2(dataset_ops.DatasetSource):
```

Comment:
Out of sheer curiosity: why V2?

Reply:
Normally in TensorFlow it looks like [this](https://github.com/tensorflow/tensorflow/blob/r1.14/tensorflow/python/data/experimental/ops/random_ops.py#L32). V2 will be exported to TensorFlow 2.0 and V1 to 1.x

For now I renamed V1 to `DALIDataset` to make the whole thing easier. When we will test it with TF 2.0 we will need V2.